lib/hanami/cli/generators/app/action/action.erb

```diff
@@ -1,12 +1,12 @@
-# auto_register: false
 # frozen_string_literal: true
 
-require "<%= underscored_slice_name %>/action"
-
-module <%= classified_slice_name %>
+module <%= classified_app_name %>
   module Actions
 <%= module_controller_declaration %>
-<%= module_controller_offset %>class <%= classified_action_name %> < <%= classified_slice_name %>::Action
+<%= module_controller_offset %>class <%= classified_action_name %> < <%= classified_app_name %>::Action
+<%= module_controller_offset %>  def handle(*, res)
```

Comment:
@timriley Not binding `req`/`res` when not needed it's faster.

Reply:
@jodosha Is it really _meaningfully_ faster? Are we willing to do that at the cost of discoverability here?

Reply:
I agree about including both request and response in the default.

Can some Rubocop rule can automatically fix things like that?

Reply:
@timriley +100 req/s.

Reply:
Additionally, I'd really like to see them spelled out as `def handle(request, response)` but I guess that is unpopular.

`req` and `res` are only one character apart, so they're easy to mess up. And, while *we're* used to it now, `res` especially isn't a common abbreviation for `response`. In context, sure, one can figure it out but it's not as obvious.

From a documentation perspective, I was thinking we could consistently use `request` and `response` for clarity, as well. But we should be consistent so users don't get misnamed variable errors.

(I've held onto this feedback for a year or so because it felt irrelevant to the end-user experience but now it's very relevant with a generator 😅 )

Reply:
To give us something we can release this week, I'm inclined to change this to:

```
def handle(*, response)
```

as a starting point.

I'm definitely convinced by the value of using the full words here (which is _much_ more common than the `req`/`res` abbreviations used here so far).

We can then have a separate discussion about the discoverability of the first `request` parameter versus any performance improvements gained from splatting it out.